baybe/kernels/priors/basic.py

```diff
@@ -0,0 +1,16 @@
+"""A Gamma prior parameterized by concentration and rate."""
```

Comment:
Module docstring should be generic, not just about gamma priors

Reply:
Changed, please resolve if happy.